src/main/java/heronarts/lx/LXEngine.java

```diff
@@ -817,7 +817,7 @@ private void sleepNanos(long nanoDuration) throws InterruptedException {
           sleep(1);
         } else {
           if (timeLeft > SPIN_YIELD_PRECISION) {
-            yield();
+            Thread.yield();
```

Comment:
It wouldn't compile unless I did this. Is there a reason `Thread.` is currently omitted?

Reply:
Weird, don't know why it would refuse to compile, maybe you have a strict setting on. But I'm also surprised my IDE didn't complain. Fixed here: https://github.com/heronarts/LX/commit/441bbb02433ed4ed886005bb85eb341c24f90de4